docs/resources/directory.md.erb

```diff
@@ -13,7 +13,7 @@ Use the `directory` InSpec audit resource to test if the file type is a director
 A `directory` resource block declares the location of the directory to be tested, and then one (or more) matchers:
 
     describe directory('path') do
-      it { should MATCHER 'value' }
+      it { should cmp 'value' }
```

Comment:
There aren't any top-level matchers for the `directory` resource that return a directly-comparable value, so we should add a property into this example:

```ruby
its('property') { should cmp 'value' }
```